app/src/pages/ODD/ProtocolSetup/index.tsx

```diff
@@ -185,7 +190,18 @@ function PrepareToRun({
     protocolRecord?.data.metadata.protocolName ??
     protocolRecord?.data.files[0].name ??
     ''
-
+  const robotType = useRobotType(robotName)
+  const { reportCameraEnablementSettings, reportPhotoAccessUsage } =
+    useCameraAnalytics({
+      source: SOURCE_RUN_RECORD,
+      robotType: robotType,
+    })
+  useEffect(() => {
+    reportPhotoAccessUsage({
+      action: 'storageWarning',
+      transactionId: runId,
+    })
+  })
```

Comment:
```suggestion
  useEffect(() => {
    reportPhotoAccessUsage({
      action: 'storageWarning',
      transactionId: runId,
    })
  }, [storageInfo.isImageStorageLow != null])
```

Note that we should prop drill `storageInfo` from `<ProtocolSetup />` into `<PrepareToRun />`